public/scripts/scripts.js

```diff
@@ -76,8 +75,7 @@ function generateTrackList(tracks){
       spanText: "four weeks"}
     trackBatch.push(track);
   });
-  return(developPlayListStats(allCallSongs, trackBatch));
-  $('#controlPanel').show();
```

Comment:
this line was never hit so i deleted it - you may want to re-add it, maybe put it above the `return` line?